qa/integration-tests/src/test/java/io/camunda/it/utils/BrokerWithCamundaExporterITInvocationProvider.java

```diff
@@ -124,18 +159,49 @@ public void beforeAll(final ExtensionContext context) {
                           .withAdditionalProperties(additionalProperties)
                           .withAdditionalProfiles(additionalProfiles)
                           .start();
-                  closeables.add(testBroker);
-                  testBrokers.put(entry.getKey(), testBroker);
+                  CLOSEABLES.add(testBroker);
+                  TEST_BROKERS.put(entry, testBroker);
                   testBroker.awaitCompleteTopology();
                   final var zeebeClientTestFactory =
                       new ZeebeClientTestFactory().withUsers(registeredUsers);
-                  zeebeClientTestFactories.put(entry.getKey(), zeebeClientTestFactory);
-                  closeables.add(zeebeClientTestFactory);
+                  zeebeClientTestFactories.put(entry, zeebeClientTestFactory);
+                  CLOSEABLES.add(zeebeClientTestFactory);
+                }
+                case RDBMS_EXPORTER_H2 -> {
+                  final var testBroker =
+                      new TestStandaloneBroker()
+                          .withBrokerConfig(cfg -> cfg.getGateway().setEnable(true))
+                          .withBrokerConfig(additionalBrokerConfig)
+                          .withRecordingExporter(true)
+                          .withProperty("camunda.rest.query.enabled", true)
+                          .withProperty("camunda.database.type", "rdbms")
+                          .withProperty(
+                              "spring.datasource.url",
+                              "jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;MODE=PostgreSQL")
+                          .withProperty("spring.datasource.driver-class-name", "org.h2.Driver")
+                          .withProperty("spring.datasource.username", "sa")
+                          .withProperty("spring.datasource.password", "")
+                          .withProperty("logging.level.io.camunda.db.rdbms", "DEBUG")
+                          .withProperty("logging.level.org.mybatis", "DEBUG")
+                          .withExporter("rdbms", cfg -> cfg.setClassName("-"))
```

Comment:
For CamundaExporter this config happens in TestStandaloneBroker#withCamundaExporter.

Don't know which location is the best but we should use the same schema for both